src/evo/creditpool.cpp

```diff
@@ -215,37 +215,17 @@ CCreditPoolManager::CCreditPoolManager(CEvoDB& _evoDb)
 {
 }
 
-CCreditPoolDiff::CCreditPoolDiff(CCreditPool starter, const CBlockIndex *pindex, const Consensus::Params& consensusParams) :
+CCreditPoolDiff::CCreditPoolDiff(CCreditPool starter, const CBlockIndex *pindex, const Consensus::Params& consensusParams, const CAmount blockSubsidy) :
     pool(std::move(starter)),
     pindex(pindex),
     params(consensusParams)
 {
     assert(pindex);
-}
-
-void CCreditPoolDiff::AddRewardRealloced(const CAmount reward) {
-    assert(MoneyRange(reward));
-    platformReward += reward;
-}
 
-bool CCreditPoolDiff::SetTarget(const CTransaction& tx, const CAmount blockSubsidy, TxValidationState& state)
-{
-    CCbTx cbTx;
-    if (!GetTxPayload(tx, cbTx)) {
-        return state.Invalid(TxValidationResult::TX_CONSENSUS, "bad-cbtx-payload");
+    if (llmq::utils::IsMNRewardReallocationActive(pindex)) {
+        // We consider V20 active if mn_rr is active
+        platformReward = MasternodePayments::PlatformShare(GetMasternodePayment(pindex->nHeight, blockSubsidy, /*v20_active=*/ true));
```

Comment:
@knst note the name of the param, not just whitespaces
```suggestion
        platformReward = MasternodePayments::PlatformShare(GetMasternodePayment(pindex->nHeight, blockSubsidy, /*fV20Active=*/ true));
```